test/allowlist_for_publicAPI.json

```diff
@@ -896,6 +896,7 @@
     "LongTensor",
     "ShortTensor",
     "addmm",
+    "solve",
```

Comment:
Adding new entry to this file is not ok. Please make sure your API is properly exposed.

Reply:
I see what's going on. Problem fixed in the commit below.